### PR TITLE
covid2019

### PR DESCRIPTION
**Pull request by satishgarimella**
please wash your hands

### CODE REVIEW

**Comment by shobancs** [comment]
Thanks for recommendations!! Good job!

**Comment by annekalyan** [comment]
Good.

File: covid.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+dsdsdsds :wq!
```
Comment (shobancs) [inline covid.txt:1]: This is balablaa , please change it Covid logic

Comment (shobancs) [inline covid.txt:1]: This is balablaa , please change it Covid logic

**Review by shobancs — COMMENTED** [review]
Hello please look at my review comments

**Review by rkolagani — COMMENTED** [review]
test test test

File: covid.txt (hunk 1 above)
Comment (satishgarimella) [reply covid.txt:1]:
```suggestion
dsdsdsds :wq! sorry guys .... 
lets have ice cream 
```

Comment (shobancs) [reply covid.txt:1]: Good job

**Comment by rkolagani** [comment]
Me got email also

**Review by shobancs — COMMENTED** [review]
Please go ahead and merge

**Pull request merged**